app/src/main/java/org/oppia/android/app/home/recentlyplayed/PromotedStoryListAdapter.kt

```diff
@@ -3,13 +3,16 @@ package org.oppia.android.app.home.recentlyplayed
 import android.view.LayoutInflater
 import android.view.ViewGroup
 import androidx.recyclerview.widget.RecyclerView
-import org.oppia.android.databinding.OngoingStoryCardBinding
+import org.oppia.android.databinding.RecentlyPlayedStoryCardBinding
 import org.oppia.android.databinding.SectionTitleBinding
 
 private const val VIEW_TYPE_SECTION_TITLE_TEXT = 1
 private const val VIEW_TYPE_SECTION_STORY_ITEM = 2
 
-/** Adapter to inflate different items/views inside [RecyclerView] for Ongoing Story List. */
+/** Adapter to inflate different items/views inside [RecyclerView] for Ongoing Story List.
+ *
+ * @param [itemList] list of items that may be displayed in recently-played fragment recycler view.
```

Comment:
```suggestion
 * @property itemList list of items that may be displayed in recently-played fragment recycler view
```

Three things:
1. We use 'property' instead of 'param' for constructor vals.
2. We should reference properties and parameters directly by name rather than using link syntax.
1. The first sentence fragment for properties and parameters don't end in punctuation.

Ditto elsewhere in the PR (for all three).

Reply:
Just to clarify that you meant in (3) that we don't punctuate if there is only one line for properties and parameters?

I've made the changes.

Reply:
Thanks! And your interpretation is correct. We only punctuate when there's more than one sentence for parameters and properties, otherwise we leave it off.